tools/releaseTools.psm1

```diff
@@ -189,4 +189,62 @@ function Get-ChangeLog
     $new_commits | Sort-Object -Descending -Property IsBreakingChange | ForEach-Object -MemberName ChangeLogMessage
 }
 
-Export-ModuleMember -Function Get-ChangeLog
+##############################
+#.SYNOPSIS
+#Gets packages which have newer packages in nuget.org
+#
+#.PARAMETER Path
+#The path to check for csproj files with packagse
+#
+#.OUTPUTS
+#Objects which represet the csproj package ref, with the current and new version
+##############################
+function Get-NewPackage
+{
+    param(
+        [String]
+        $Path = (Join-path -Path $PSScriptRoot -ChildPath '..')
+    )
+    # Calculate the filter to find the CSProj files
+    $path = Join-path -Path $PSScriptRoot -ChildPath '..'
+    $filter = Join-Path -Path $path -ChildPath '*.csproj'
+    $csproj = Get-ChildItem $filter -Recurse
+
+    $csproj | ForEach-Object{
+        $file = $_
+
+        # parse the csproj
+        [xml] $csprojXml = (Get-content -Raw -Path $_)
+
+        # get the package references
+        $packages=$csprojXml.Project.ItemGroup.PackageReference
+
+        # check to see if there is a newer package for each refernce
+        foreach($package in $packages)
+        {
+            # Get the name of the package
+            $name = $package.Include
+
+            # don't pull 'Microsoft.Management.Infrastructure' from nuget
+            if($name -and $name -ne 'Microsoft.Management.Infrastructure')
+            {
+                # Get the current package from nuget
+                $newPackage = find-package -Name $name -Source https://nuget.org/api/v2/  -ErrorAction SilentlyContinue
```

Comment:
'donet add package' apparently handles updating nuget packages as well. We should not re-invent the wheel.  https://ardalis.com/how-do-i-update-a-nuget-package-using-dotnet-cli